var/spack/repos/builtin/packages/singularityce/package.py

```diff
@@ -79,6 +77,8 @@ def edit(self, spec, prefix):
                 confstring += " --without-suid"
             if "~network" in spec:
                 confstring += " --without-network"
+            if self.singularity_name != "apptainer":
+                confstring += " --without-conmon"
```

Comment:
Does this mean that `conmon` is an optional dependency that should be behind a variant?

Reply:
For `singularityce` I think `conmon` is needed but it can either come from the system or a submodule. I think for `spack` the way to go in those cases is to use the package coming from spack and this is why I did that change.
For `apptainer` I don't see any reference of `conmon` in the repository so I guess it's not needed and this is why I don't append the ` --without-conmon` in the configuration.